### PR TITLE
[14.0][MIG] shopfloor_dangerous_goods

### PR DESCRIPTION
**Pull request by mmequignon**
Depends on:
- https://github.com/OCA/stock-logistics-workflow/pull/937

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 freezegun
 odoo_test_helper
+
+odoo14-addon-stock_dangerous_goods @ git+https://github.com/OCA/stock-logistics-workflow@refs/pull/937/head#subdirectory=setup/stock_dangerous_goods
```
Comment (sebalix) [inline test-requirements.txt]: Add also all other dependencies (`l10n_eu_product_adr` etc) to get tests running on Travis.

Comment (mmequignon) [reply test-requirements.txt]: done

**Comment by simahawk** [comment]
I'm removing the test dep here to avoid conflicts w/ other PRs.
@mmequignon pls get back to this ASAP :pray:

File: shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.detail-picking-select .has-lq-products .v-list-item__title::after {
```
Comment (simahawk) [inline shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css:1]: Note to myself: this selector is compliant w/ a change that I still have to push in another PR

**Review by simahawk — CHANGES_REQUESTED** [review]
blocking for me

File: shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css (hunk 2 above)
Comment (simahawk) [reply shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css:1]: Here's the change https://github.com/OCA/wms/pull/582

Comment (simahawk) [reply shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css:1]: merged, rebased, we are good to go

**Review by simahawk — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by simahawk** [comment]
@mmequignon a great day for humanity.... :stuck_out_tongue_winking_eye: 

/ocabot merge nobump

**Pull request merged**